Homework01/task1_11/src/test/java/by/bsu/tat/main/BicycleTest.java

```diff
@@ -0,0 +1,40 @@
+package by.bsu.tat.main;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.Assert.*;
+
+/**
+ * @author Alexey Makovski.
+ */
+
+public class BicycleTest {
```

Comment:
You have ignored my comment but updated document that the code is ready for review. Review won't be performed until you answer to the next question or fix the code:
1/ On which route these tests are performed? If the one that is in "Coordinates.txt" this is not acceptable. Route must be specified here in the test.